### ISSUE TITLE
When changing the Order of a List no publish event is raised. (using Headless CMS)

### ISSUE THREAD

**Issue by HarutyunGazyan** [OrchardCMS/OrchardCore#11106]
![image](https://user-images.githubusercontent.com/48087308/151799261-5c864b3a-a4c7-4c4e-b123-1f216886908b.png)
When changing the content item order here, 
![image](https://user-images.githubusercontent.com/48087308/151799466-a3073dc8-3a4c-4cf7-a34f-ba8825bd54cf.png)
overridden Task PublishedAsync(PublishContentContext context) method is not being called.
App only calls UpdatingAsync and UpdatedAsync methods from the same class.
But in GraphQL request I can see that the change of order took the place, so it has been published in fact.
Also in GraphQL API it's not possible to query the Order property of ContainedPart.
![image](https://user-images.githubusercontent.com/48087308/151800083-62d18601-871a-4dac-98b4-bb383e5bda58.png)
Basically what I do, is that I catch every publish/unpublish event, then send it via ServiceBus, then receiver is receiving that event, queries the CMS via GraphQL and get the data based on the ID from that event.

**Comment by HarutyunGazyan** [comment]
Note: this is how Ordering is set up, still can not query the order frim GraphQL
![image](https://user-images.githubusercontent.com/48087308/151801845-5b94f91b-29af-4038-b859-152e42df24bd.png)

**Comment by deanmarcussen** [comment]
Yes the Order property should be exposed to graphql 

would you do a Pr for it @HarutyunGazyan 

tye published event is not fired when the order is changed because the content item is not drafted and published. I think the update handler should be called. Can you check that?

**Comment by HarutyunGazyan** [comment]
> Yes the Order property should be exposed to graphql
> 
> would you do a Pr for it @HarutyunGazyan
> 
> tye published event is not fired when the order is changed because the content item is not drafted and published. I think the update handler should be called. Can you check that?
Order should be exposed, but it is not
as you can see here
![image](https://user-images.githubusercontent.com/48087308/151852653-e46a46ea-9221-44cb-9396-803921fa3122.png)
Update handler is called, can you also please list else when Publish handler is not called?
Order doesnt exposed even in Menu graphQL

**Comment by sebastienros** [comment]
@HarutyunGazyan do you want to submit a PR adding the property to GraphQL? Looking at how other properties are added should help.

**Comment by gvkries** [comment]
Note, the order property has since been exposed in GraphQL. The publish event is still not raised when changing the order.
